src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,137 @@
+package seedu.address.model.event;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import seedu.address.model.particpant.Participant;
+
+/**
+ * This is an Event class.
+ */
+public class Event implements Comparable<Event> {
+
+    private boolean isDone = false;
+    private ArrayList<Participant> participants = new ArrayList<>();
+    private EventName eventName;
+    private EventDate eventDate;
```

Comment:
I was wondering if it would be possible to combine both date and time together like Java's LocalDateTime? Would that make it easier

Reply:
I split it for now, since it will be easier to test. I think since we making the "time" field optional, it will be better to split? I am not too familiar 😅 . I guess @davelowqx  when implementing addEvent can decide which is easier and make changes accordingly.